src/Symfony/Component/Messenger/Bridge/Beanstalkd/Tests/Transport/ConnectionTest.php

```diff
@@ -503,3 +517,7 @@ public function testSendWithRoundedDelay()
         $connection->send($body, $headers, $delay);
     }
 }
+
+interface PheanstalkInterface extends PheanstalkPublisherInterface, PheanstalkSubscriberInterface, PheanstalkManagerInterface
```

Comment:
The `PheanstalkInterface` was split into multiple interfaces. Since the client implements all of them, it's hard to mock. This is a helper interface to make mocking easier.